### PR TITLE
Quote externalId and version when calling workflows.executions.run(...)

### PR DESCRIPTION
**Pull request by VerstraeteBert**
## Checklist:
- [x] Tests added/updated.
- [x] Documentation updated. Documentation is generated from docstrings - these must be updated according to your change.
  If a new method has been added it should be referenced in [cognite.rst](https://github.com/cognitedata/cognite-sdk-python/blob/master/docs/source/cognite.rst) in order to generate docs based on its docstring.
- [x] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/cognite-sdk-python/blob/master/CHANGELOG.md).
- [x] Version bumped. If triggering a new release is desired, bump the version number in [_version.py](https://github.com/cognitedata/cognite-sdk-python/blob/master/cognite/client/_version.py) and [pyproject.toml](https://github.com/cognitedata/cognite-sdk-python/blob/master/pyproject.toml) per [semantic versioning](https://semver.org/).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [7.63.8] - 2024-10-22
+### Fixed
+- Data Workflows: can now use a workflowExternal and/or version containing a `/` when calling `workflows.executions.run`
```
Comment (haakonvt) [inline CHANGELOG.md:22]:
```suggestion
- Data Workflows: workflow external ID and version are now URL encoded to allow characters like `/`  when calling `workflows.executions.run`
```

File: cognite/client/_api/workflows.py [hunk 2]
```diff
@@ -396,7 +396,9 @@ def run(
         if metadata is not None:
             body["metadata"] = metadata
 
-        response = self._post(url_path=f"/workflows/{workflow_external_id}/versions/{version}/run", json=body)
+        response = self._post(
+            url_path=f"/workflows/{quote(workflow_external_id, '')}/versions/{quote(version, '')}/run", json=body
```
Comment (haakonvt) [inline cognite/client/_api/workflows.py:400]: FYI, there exist a utility function for this already, `interpolate_and_url_encode`

File: CHANGELOG.md [hunk 3]
```diff
@@ -17,6 +17,10 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [7.63.8] - 2024-10-22
```
Comment (haakonvt) [inline CHANGELOG.md:20]: In case you want to merge immediately 😉 
```suggestion
## [7.63.8] - 2024-10-21
```

**Review by haakonvt — APPROVED** [review]
LGTM, a few comments:

File: cognite/client/_api/workflows.py (hunk 2 above)
Comment (VerstraeteBert) [reply cognite/client/_api/workflows.py:400]: Yes, the other methods are using the quoting method. I'd prefer to keep the method in the file the same.

Comment (haakonvt) [reply cognite/client/_api/workflows.py:400]: I leave that up to you 😊

**Pull request merged**